lib/saml_idp_metadata/parser.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-require 'active_support/dependencies/autoload'
+require 'active_support'
 require 'active_support/core_ext'
```

Comment:
Before Code was error occurred.
```
An error occurred while loading spec_helper.
Failure/Error: require 'active_support/core_ext'

NameError:
  uninitialized constant ActiveSupport::Deprecation
```

Modify following the Rails guide.
https://guides.rubyonrails.org/active_support_core_extensions.html#loading-all-core-extensions